core/src/test/java/org/apache/calcite/test/SqlValidatorTest.java

```diff
@@ -803,7 +804,9 @@ public void _testConvertAndTranslate() {
     checkWholeExpFails("translate('aabbcc', 'ab', '+-')",
         "No match found for function signature TRANSLATE3\\(<CHARACTER>, <CHARACTER>, <CHARACTER>\\)");
     tester = tester.withOperatorTable(
-        ChainedSqlOperatorTable.of(OracleSqlOperatorTable.instance(),
+        ChainedSqlOperatorTable.of(SqlDialectOperatorTableFactory
+                .instance()
+                .getOperatorTable(SqlDialect.Dialect.ORACLE),
```

Comment:
Please break the line after "of(" to make it clear what ".instance()" applies to.

Reply:
I agree, will format